Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -105,6 +105,10 @@ public static IServiceCollection AddJellyfinApiSwagger(this IServiceCollection s
                 {
                     c.IncludeXmlComments(xmlFile);
                 }
+
+                // Order actions by route path, then by http method.
+                c.OrderActionsBy(description =>
+                    $"{description.ActionDescriptor.RouteValues["controller"]}_{description.HttpMethod}");
```

Comment:
Can you expand on what this changes? This seems sensible to me, but what was the original ordering?

Reply:
The original ordering was whatever order the routes were created in the controller. Ideally they would also be created in the "correct" order, but this forces the swagger to generate it ordered.

I found this when I was reading up on swashbuckle: https://github.com/domaindrivendev/Swashbuckle.AspNetCore#change-operation-sort-order-eg-for-ui-sorting

Reply:
Cool, makes sense to me thanks! Ideally we would have a sensible ordering in our controllers but that seems like a real pain to decide on a standard for, code review, etc. So 👍 to this.